src/main/scala/cromwell/engine/backend/sge/SgeBackendCall.scala

```diff
@@ -25,4 +25,7 @@ case class SgeBackendCall(backend: SgeBackend,
   override def execute(implicit ec: ExecutionContext) = backend.execute(this)
 
   override def poll(previous: ExecutionHandle)(implicit ec: ExecutionContext) = Future.successful(previous)
+
+  override def useCachedCall(avoidedTo: BackendCall)(implicit ec: ExecutionContext): Future[ExecutionHandle] =
+    backend.useCachedCall(avoidedTo.asInstanceOf[SgeBackendCall], this)
```

Comment:
I'd like to avoid `instanceof`s if possible, but given how dynamic our use of backends is going to be I'm not sure we can do a lot better than this

i.e. I don't think `collectFirst { case s: SgeBackendCall => s } get` would be much better.